main/autogen/subclasses.cc

```diff
@@ -92,34 +95,63 @@ optional<Subclasses::SubclassInfo> Subclasses::descendantsOf(const Subclasses::M
     return SubclassInfo(fnd->second.classKind, std::move(out));
 }
 
+const core::SymbolRef Subclasses::getConstantRef(core::GlobalState &gs, string rawName) {
+    core::UnfreezeNameTable nameTableAccess(gs);
+    core::UnfreezeSymbolTable symbolTableAccess(gs);
+    core::ClassOrModuleRef sym = core::Symbols::root();
+
+    for (auto &n : absl::StrSplit(rawName, "::")) {
+        sym = gs.enterClassSymbol(core::Loc(), sym, gs.enterNameConstant(gs.enterNameUTF8(n)));
```

Comment:
What do you think about using `const core::GlobalState &gs` everywhere, and using `lookupClassSymbol` everywhere?

This code all seems to me like it shouldn't need mutable access to GlobalState.

Reply:
Sounds good, just added a change to make it constant everywhere